hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -439,44 +459,63 @@ static void add_stage_code(struct PixelShader *ps,
     QString *sum_dest = get_var(ps, output.muxsum, true);
 
     if (qstring_get_length(ab_dest)) {
-        qstring_append_fmt(ps->code, "%s.%s = %s(%s);\n",
-                           qstring_get_str(ab_dest), write_mask, caster, qstring_get_str(ab_mapping));
+        qstring_append_fmt(ps->code,
+                           "%s.%s = %s(%s);\n",
+                           qstring_get_str(ab_dest),
+                           write_mask,
```

Comment:
I'd like to have the writemask on the same line as `ab_dest` because they have a *very* strong relation.